plugins/custom_calyptia/calyptia.c

```diff
@@ -96,13 +96,13 @@ static void pipeline_config_add_properties(flb_sds_t *buf, struct mk_list *props
             flb_sds_printf(buf, "    %s ", kv->key);
 
             if (is_sensitive_property(kv->key)) {
-                flb_sds_printf(buf, "--redacted--");
+                flb_sds_cat(*buf, "--redacted--", strlen("--redacted--"));
```

Comment:
can you use flb_sds_cat_safe() version ?, that version will take care if a realloc happens and the address change.

Reply:
Done!